app/lib/bk/compat/parsers/bitbucket.rb

```diff
@@ -67,6 +54,14 @@ def parse
 
       private
 
+      def register_translators
+        BK::Compat::BitBucketSteps::Import.new(register: method(:register_translator))
+        BK::Compat::BitBucketSteps::Parallel.new(register: method(:register_translator), recursor: method(:translate_step))
+        BK::Compat::BitBucketSteps::Stages.new(register: method(:register_translator), recursor: method(:translate_step))
+        BK::Compat::BitBucketSteps::Step.new(register: method(:register_translator), definitions: @config.fetch('definitions', {}))
```

Comment:
I'd suggest keeping the original formatting to avoid lines being way too long:
```suggestion
        BK::Compat::BitBucketSteps::Parallel.new(
          register: method(:register_translator),
          recursor: method(:translate_step)
        )
        BK::Compat::BitBucketSteps::Stages.new(
          register: method(:register_translator),
          recursor: method(:translate_step)
        )
        BK::Compat::BitBucketSteps::Step.new(
          register: method(:register_translator),
          definitions: @config.fetch('definitions', {})
        )
```

Reply:
Have reverted to the multiline way (agreed that its easier on the eyes!)